src/templates/errors.html

```diff
@@ -4,8 +4,8 @@
   <p class="help-block" ng-show="formioForm[componentId].$error.number">{{ component.label || component.placeholder || component.key }} {{'must be a number' | formioTranslate:null:builder}}.</p>
   <p class="help-block" ng-show="formioForm[componentId].$error.maxlength">{{ component.label || component.placeholder || component.key }} {{'must be shorter than' | formioTranslate:null:builder}} {{ component.validate.maxLength + 1 }} {{'characters' | formioTranslate:null:builder}}.</p>
   <p class="help-block" ng-show="formioForm[componentId].$error.minlength">{{ component.label || component.placeholder || component.key }} {{'must be longer than' | formioTranslate:null:builder}} {{ component.validate.minLength - 1 }} {{'characters' | formioTranslate:null:builder}}.</p>
-  <p class="help-block" ng-show="formioForm[componentId].$error.min">{{ component.label || component.placeholder || component.key }} {{'must be higher than' | formioTranslate:null:builder}} {{ component.validate.min - 1 }}.</p>
```

Comment:
@travist I'm going to change both, so they say:

`must be greater than or equal to x`
`must be less than or equal to x`

any objections?